core/src/main/java/nl/nn/adapterframework/http/rest/ApiListenerServlet.java

```
@@ -557,6 +567,20 @@ else if(segment.startsWith("{") && segment.endsWith("}")) {
 		}
 	}
 
+	private Message parseContentAsMessage(InputStream inputStream, String contentType) {
+		String charset = null;
+		if(StringUtils.isNotEmpty(contentType)) {
+			try {
+				ContentType parsedContentType = ContentType.parse(contentType);
+				Charset parsetCharset = parsedContentType.getCharset();
+				charset = parsetCharset.displayName();
+			} catch (Exception e) {
+				//Silently fail if we cannot parse the charset.
```

waarom silently failen?

---

Tja, kan er inderdaad beter een log melding bij zetten